integration-tests/elasticsearch-rest-high-level-client/README.md

```diff
@@ -1,4 +1,4 @@
-# quarkus-integration-test-elasticsearch project
+# quarkus-integration-test-elasticsearch-rest-client project
```

Comment:
Aren't you missing `high-level` here and below? Not sure as GH UI is doing weird things with those files but better check.

Reply:
Sorry, bad copy/paste. Fixed.